src/pg_to_evalscript/javascript_processes/any.js

```diff
@@ -0,0 +1,38 @@
+function any(arguments) {
+  const { data, ignore_nodata = true } = arguments;
+
+  if (data === undefined) {
+    throw new Error("Mandatory argument `data` is not defined.");
+  }
+
+  if (!Array.isArray(data)) {
+    throw new Error("Argument `data` is not an array.");
+  }
+
+  if (!data.every((x) => typeof x === "boolean" || x === null)) {
+    throw new Error(
+      "Values in argument `data` can only be of type boolean or null."
+    );
+  }
+
+  if (ignore_nodata) {
+    if (data.filter((x) => x !== null).length === 0) {
+      return null;
+    }
+    return data.filter((x) => x !== null).some((x) => x === true);
+  }
+
+  if (data.length === 0) {
+    return null;
+  }
```

Comment:
Same as above - if I understand correctly, this can be optimized, so that this `if` and  `if (data.filter((x) => x !== null).length === 0) {` are before `if (ignore_nodata) {` since the behaviour is the same regardless of `ignore_nodata` value.
(cases when the array is empty or when all elements in the array are null)

Reply:
Should be fixed.